MD_Simulations_on_Local_Machine.md

```diff
@@ -0,0 +1,49 @@
+# Standard Operating Procedure:  
+## Running Molecular Dynamic Simulations on your Local Machine
+
+### 1. Purpose
+- This SOP outlines the steps necessary to run Molecular Dynamic Simulations using OpenMM and OpenFF on your local machine.
+
+### 2. Procedure
+##### 2.1 Downloading Necessary Software
+###### 2.1.1 Install Visual Studio Code
+- Follow the instructions to download VS Code [here](https://code.visualstudio.com/Download)
```

Comment:
1. Do you need so many layers of headings?
2. If you do use headings, use them in sequence, and don't skip heading levels.